scripts/common/rook.sh

```diff
@@ -204,13 +204,45 @@ function rook_ceph_to_sc_migration() {
     report_addon_success "rook-ceph-to-$scProvisioner-migration" "v2"
 }
 
-# if PVCs and object store data have both been migrated from rook-ceph and rook-ceph is no longer specified in the kURL spec, remove rook-ceph
 function maybe_cleanup_rook() {
     if [ -z "$ROOK_VERSION" ]; then
+
+        if kubectl get ns | grep -q rook-ceph; then
+            logStep "Removing Rook ..."
+        fi
```

Comment:
Two things
1. no `...`
2. put this log line inside `if [ "$DID_MIGRATE_ROOK_PVCS" == "1" ] && [ "$DID_MIGRATE_ROOK_OBJECT_STORE" == "1" ]; then`